codice/ddf
# DDF-2105: Adds registry-report-viewer

#### What does this PR do?
Creates a report for a given registry metacard to display information about that node.

#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@clockard @mcalcote @gordocanchola @brianfelix 
@djblue @andrewkfiedler @tbatie 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@shaundmorris 
@pklinef
#### How should this be tested?
This can be tested by following these steps:
1. Installing the registry app
2. Ingesting a registry metacard `ingest -t rim:RegistryPackage /Users/PathToYourDDF/ddf/catalog/spatial/registry/registry-ebrim-transformer/src/test/resources/csw-full-registry-package.xml`
3. Ingesting a second non-registry metacard
4. Open the search UI and do a wildcard search, the use the Details option to search for a metacard-tags contains registry search
5. Select the registry metacard returned by this second search and select the Action tab
6. Observe the URL generated when selecting Export Registry Metacard Information -> Display in new tab/window
7. Verify that the report is populated
#### Any background context you want to provide?
https://github.com/codice/ddf/pull/794 
Uses this action provider. 
#### What are the relevant tickets?
DDF-2105, DDF-2008, DDF-2106
#### Screenshots (if appropriate)
#### Checklist:
